src/app/config.cc

```diff
@@ -98,10 +98,10 @@ void Configuration::ParseKovriConfig() {
     // 4 = debug info warn error fatal
     // 5 = trace debug info warn error fatal
     ("log-level", bpo::value<std::uint16_t>()->default_value(3))
-    ("kovriconf,c", bpo::value<std::string>(&kovri_config)->default_value(
-        (kovri::core::GetConfigPath() / "kovri.conf").string()))
-    ("tunnelsconf,t", bpo::value<std::string>(&tunnels_config)->default_value(
-        (kovri::core::GetConfigPath() / "tunnels.conf").string()));
+    ("kovriconf,c", bpo::value<std::string>()->default_value(
+        "$DATA-DIR/config/kovri.conf"))
+    ("tunnelsconf,t", bpo::value<std::string>()->default_value(
+        "$DATA-DIR/config/tunnels.conf"));
```

Comment:
>"$DATA-DIR"

These took me aback for a moment as they were misleading.

Since these are now just placeholders, I think we should simply initialize with an empty string.